lively.collab/comments/commentBrowser.js

```diff
@@ -31,16 +31,32 @@ export class CommentBrowser extends Window {
     await instance.addCommentForMorph(comment, morph);
   }
 
-  static async initializeCommentBrowser () {
-    const firstTimeOpened = !instance;
+  static initializeCommentBrowser () {
     new CommentBrowser();
-    if (firstTimeOpened) {
-      await instance.initializeCommentGroupMorphs();
+  }
+
+  static open () {
+    if (!instance || CommentBrowser.isOpen()) return;
```

Comment:
Should the !instance case forward to initializeCommentBrowser?

Reply:
Sounds good to me